docker-compose.yml

```diff
@@ -80,6 +80,19 @@ services:
       - ./local-config/ipfs/volumes/ipfs-node-0:/data/ipfs
       - ./local-config/ipfs/private-ipfs-network/swarm.key:/data/ipfs/swarm.key
       - ./local-config/ipfs/private-ipfs-network/init.sh:/usr/local/bin/start_ipfs
+  postgres:
+    # See https://hub.docker.com/_/postgres/ for config info, e.g. how to run
+    # initialization scripts using an image derived from the official image.
+    image: postgres:14.1-alpine3.14
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
```

Comment:
I noticed that we favor literal values in the docker-compose.yml for settings other than username and password. Would you be able to set the port number to a literal value? Granted this is a small change but does align us with the current approach. 

What are you thoughts on the current state vs using the .env for "ALL" settings?

Reply:
Sure.  That's fine.  I'll make the port literal.

Reply:
done.  I also made the database name a literal, so just the username and password is in .env.